cfgov/unprocessed/js/molecules/Multiselect.js

```diff
@@ -21,6 +21,14 @@ var strings = require( '../modules/util/strings' );
  */
 function Multiselect( element ) { // eslint-disable-line max-statements, inline-comments, max-len
 
+  var BASE_CLASS = 'cf-multi-select';
+
+  // TODO: Add 'cf-multi-select' class to <select> menu markup
```

Comment:
On second thought, should we be using a js-hook rather than a classname? We don't want the styles applied to the native select.